packages/stylelint/css-modules/index.mjs

```diff
@@ -24,6 +24,8 @@ export default {
       },
     ],
 
+    'declaration-property-value-no-unknown': true,
```

Comment:
https://github.com/ijlee2/embroider-css-modules/blob/2.0.19/docs/written-guides/refactor-code.md#use-var-for-variables